### ISSUE TITLE
Storage not working (This backend doesn't support absolute paths.)

### ISSUE THREAD

**Issue by null-none** [lehins/django-smartfields#29]
```python
from storages.backends.s3boto3 import S3Boto3Storage
```
...
```python
class MediaStorage(S3Boto3Storage):
    location = 'media'
    file_overwrite = False
```
...
```python
    video = fields.FileField(storage=MediaStorage(),
...
```

Error: `This backend doesn't support absolute paths.` 

Without `DEFAULT_FILE_STORAGE` and `STATICFILES_STORAGE` working correctly and with Django ImageField and FileField working correctly. Thank you.

Where problem?

**Comment by lehins** [comment]
I wish I had a simple answer for you, but unfortunately I don't have the means to reproduce the problem, so if you figure it out please post a solution here or maybe even submit a PR with a patch if that issue can be prevented at the library level.

**Comment by null-none** [comment]
@lehins Submitted Pull Request, check please.

**Comment by lehins** [comment]
Fixed in #32 
Thanks again.
